dockerfiles/pi-opendds-cross/build.sh

```diff
@@ -5,55 +5,61 @@ OPENDDS_GIT_REPO=https://github.com/objectcomputing/OpenDDS
 OPENDDS_MAKE_JOBS=4
 OPENDDS_PREFIX=${BUILD_DIR}/pi-opendds
 
-wget -nc http://download.objectcomputing.com/OpenDDS/resources/gcc-8.2.0-rpi.tar.bz2
-tar xjf gcc-8.2.0-rpi.tar.bz2
+wget -nc -O gcc-arm-10.2-2020.11-x86_64-arm-none-linux-gnueabihf.tar.xz \
```

Comment:
Is there a way to reduce duplication between Dockerfile `RUN` script commands and the `build.sh`?

Reply:
Yes. The docker image can include and run build.sh.